packages/neon/neon/lib/src/settings/models/select_option.dart

```diff
@@ -25,9 +32,24 @@ class SelectOption<T> extends Option<T> {
     required super.defaultValue,
     required final Map<T, LabelBuilder> values,
     required super.enabled,
+
+    /// Force loading the stored value.
+    ///
+    /// This is needed when [values] is empty but the stored value should still
+    /// be loaded.
+    /// This only works when [T] is of type String?.
```

Comment:
Here it says String? while further up it says String. I assume you meant String here as well?

Reply:
`String?` is right although it's debatable wether a type definition for a generic does always implicitly include the null value as well. 

Where did I say plain `String` ?

Reply:
The plain String is in the comment in the other constructor. I'm fine with both ways, but they should say the same thing